ocs_ci/templates/CSI/rbd/pod.yaml

```diff
@@ -6,11 +6,11 @@ metadata:
   namespace: default
 spec:
   containers:
-   - name: web-server
-     image: nginx
+   - name: fio
+     image: fio
```

Comment:
instead of changing image and name in every yaml, why not change this param by getting the yaml in dict.